clang/lib/CodeGen/CoverageMappingGen.cpp

```diff
@@ -1829,6 +1833,22 @@ void CoverageMappingModuleGen::emit() {
                              llvm::GlobalValue::InternalLinkage, NamesArrVal,
                              llvm::getCoverageUnusedNamesVarName());
   }
+  const StringRef VarName(INSTR_PROF_QUOTE(INSTR_PROF_RAW_VERSION_VAR));
```

Comment:
I already reverted changes in Clang because `VARIANT_MASK_BIN_CORRELATE` flag is no longer necessary: https://github.com/llvm/llvm-project/pull/69493#issuecomment-1815324995